source/io/module_javaScriptInvoke.js

```diff
@@ -432,18 +442,33 @@
                     throw new Error('The API being accessed for ' + functionName + ' is not valid anymore.');
                 }
                 completionCallbackStatus.retrievalState = completionCallbackRetrievalEnum.RETRIEVED;
-                return generateCompletionCallback(occurrencePointer, functionName, returnTypeName, returnValuePointer, errorStatusPointer, errorCodePointer, errorSourcePointer, completionCallbackStatus);
+                return generateCompletionCallback(occurrencePointer, functionName, returnTypeName, returnValuePointer, errorStatusPointer, errorCodePointer, errorSourcePointer, completionCallbackStatus, isInternalFunction);
             };
+
+            if (isInternalFunction) {
+                api.setLabVIEWError = function (statusCode, errorCode, errorMessage) {
```

Comment:
Should keep the parameter names close to the lv error representation: `status`, `code`, `source`.

Reply:
longer term now that we have good apis for writing to clusters we will be able to reduce the number of references we pass around to just the error cluster instead of all the componenets

Reply:
done.